hello_world/hello_zero.ipynb

```diff
@@ -4,47 +4,46 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<img src=\"https://raw.githubusercontent.com/Qiskit/qiskit-tutorials/master/images/qiskit-heading.png\" alt=\"Note: In order for images to show up in this jupyter notebook you need to select File => Trusted Notebook\" width=\"500 px\" align=\"left\">"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## _*Doing nothing with Qiskit Terra*_ \n",
+    "# Updated Doing Nothing with Quiskit\n",
```

Comment:
It's Qiskit :sweat_smile:

Reply:
Omg, you're absolutely right, sorry about that